dependencies.yaml

```
@@ -666,7 +674,7 @@ dependencies:
     common:
       - output_types: [conda]
         packages:
-          - &pytorch_conda pytorch>=2.0,<2.2.0a0
+          - &pytorch_unsuffixed pytorch>=2.0,<2.2.0a0
```

Not sure if we discussed this in another PR (lost track) but we don’t need the suffix here.
```suggestion
          - &pytorch pytorch>=2.0,<2.2.0a0
```

---

we definitely did! It must have been on a different PR (maybe `wholegraph`?). I agree, will fix this up in a follow-up PR.